RobotCrawler/src/test/java/parser/ebookscom/EBooksParserTest.java

```diff
@@ -0,0 +1,88 @@
+package parser.ebookscom;
+
+import model.BookData;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import parser.BookDataCollector;
+import parser.DocumentBuilder;
+import parser.Parser;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Testing the parser and test how the collector is working, if collect the data properly.
+ */
+public class EBooksParserTest {
+
+    private Document theHtmlFile = null;
+    private Parser<Element> parser = new EBooksParser();
+    private BookDataCollector getter = new EBooksDataCollector();
+    @BeforeTest
+    protected void initTest(){
+        File file = new File(System.getProperty("user.dir") + "/src/test/resources/parser-ebook-test-1.html");
```

Comment:
You can get file here easier. For example you can use Files from nio like below
Paths.get(getClass().getResource("/parser-ebook-test-1.html").toURI()